staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -2385,6 +2385,7 @@ func RunTestGuaranteedUpdate(ctx context.Context, t *testing.T, store InterfaceW
 }
 
 func RunTestGuaranteedUpdateWithTTL(ctx context.Context, t *testing.T, store storage.Interface) {
+	t.Parallel()
```

Comment:
let's not change things here - for parallelizing stuff let's only parallelize them where they are really run